genome_designer/conf/global_settings.py

```diff
@@ -426,7 +426,7 @@ def _generate_secret_key(dest):
 
 FLAG__GENERATE_NEW_REFERENCE_GENOME_ENABLED = True
 
-FLAG__GENOME_FINISHING_ENABLED = False
+FLAG__GENOME_FINISHING_ENABLED = True
```

Comment:
Keep this as False.

To avoid accidentally committing things like this, create a local_settings.py file and put overrides there. local_settings.py is .gitignored.